packages/interfaces/src/protocols.ts

```diff
@@ -12,6 +12,15 @@ export enum Protocols {
   Filter = "filter",
 }
 
+export enum Codecs {
```

Comment:
The protocol id is purely specific to the implementation. It belongs and must remain close to the package implementing the protocol.

Anytime you see a need to move the protocol id in `interfaces`, assume it is a code smell and the problem is elsewhere.